diktat-rules/src/main/resources/diktat-analysis.yml

```
@@ -0,0 +1,229 @@
+- name: DIKTAT_COMMON
+  configuration:
+    domainName: org.cqfn.diktat
```

```suggestion
    domainName: your.package.name.here
```
This file should be an example: it will be bundled into jar and should contain settings that users will see as defaults. `diktat-analysis` in root directory is the one that will be used in our own checks.